### ISSUE TITLE
feedback on milestone #2

### ISSUE THREAD

**Issue by barryross** [allisonhall/FEWD-FINAL-PROJECT#1]
Hey Allison, nice work on the HTML for this!  Are you comfortable having a draft of the CSS done by Monday?

**Comment by allisonhall** [comment]
yes I'm sorry I missed that requirement last week - I can do Monday. To answer the other question, yes - I want only one image to come up at a time based on the nav selection. I'm still working through how to do that.

**Comment by barryross** [comment]
No problem about the milestone...glad Monday seems doable for you!  After I asked that second question, I took another look at your wireframe and realized you had already described the scroll process, which totally works too. That being said, if you are going to go with the process you just described, and have hit a wall with it, I'm happy to give you a hand.
